### ISSUE TITLE
missing peerDep of eslint-config-airbnb

### ISSUE THREAD

**Issue by amilajack** [iamturns/eslint-config-airbnb-typescript#60]
`eslint-config-airbnb` [requires a peerDep of `eslint-plugin-react-hooks`](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json#L82):

```
warning "eslint-config-airbnb-typescript > eslint-config-airbnb@18.0.1" has incorrect peer dependency "eslint-plugin-re
act-hooks@^1.7.0".
```
I'd love to submit a PR for this. Would that be welcome?

**Comment by iamturns** [comment]
Hey @amilajack - absolutely, all PRs welcome!
I think this repo needs to be broken into two separate packages to fix this issue. If so, it's quite a bit of work.
But if you find another way to solve this problem, very happy to discuss and/or review a PR 👍 
This issue is reported and tracked in #6, so will close this one.
Cheers!
